frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -81,11 +81,19 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
             <Nav className="mr-auto">
               {
                 hasRole(currentUser, "ROLE_USER") && (
+
                   <NavDropdown title="Schools" id="appnavbar-schools-dropdown" data-testid="appnavbar-schools-dropdown" >
                     <NavDropdown.Item href="/schools/list" data-testid="appnavbar-schools-list">List</NavDropdown.Item>
                     {
                       hasRole(currentUser, "ROLE_ADMIN") && (
                         <NavDropdown.Item href="/schools/create" data-testid="appnavbar-schools-create">Create</NavDropdown.Item>
+
```

Comment:
It looks like the schools and laptop navigation got mashed together when they should be separate